src/services/appDataService.ts

```diff
@@ -0,0 +1,112 @@
+import taggingUtils, { tagKeys } from '../lib/taggingUtils';
+import documentRoutes from '../routes/documentRoutes';
+import recordService from './recordService';
+import { prepareSearchParameters } from './fhirService';
+
+export interface DecryptedAppData {
```

Comment:
There seems to be some overlaps with the interface from recordservice?

Reply:
I guess the interfaces are similar, but shouldn't be. `DecryptedRecord ` should not have the `data` property